### PR TITLE
Add doc to onDemandEntries configuration

### PR DESCRIPTION
**Pull request by AugustinLF**
I didn't add docs when adding the `pagesBufferLength` option. Since `maxInactiveAge` was not documented either, this is a PR for both options.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -931,6 +931,21 @@ module.exports = {
...
 
+#### Configuring the onDemandEntries
+
+Next exposes some options that give you some control over how the server will dispose or keep in memories pages built:
+
+```js
+module.exports = {
+  onDemandEntries: {
+    // period where the server will keep pages in the buffer, defaults to 25s
+    maxInactiveAge,
```
Comment (timneutkens) [inline readme.md]: Lets do `maxInactiveAge: 25` etc 😄

Comment (AugustinLF) [reply readme.md]: I amended the commit!

File: readme.md [hunk 2]
```diff
@@ -931,6 +931,21 @@ module.exports = {
...
 
+#### Configuring the onDemandEntries
+
+Next exposes some options that give you some control over how the server will dispose or keep in memories pages built:
+
+```js
+module.exports = {
+  onDemandEntries: {
+    // period (in ms) where the server will keep pages in the buffer
+    maxInactiveAge: 25 * 1000,
```
Comment (timneutkens) [inline readme.md]: Awesome, this looks much better 💯 ❤️

**Review by timneutkens — APPROVED** [review]

**Review by timneutkens — APPROVED** [review]
Thanks! ❤️

**Pull request merged**